c4a-test/build.gradle

```diff
@@ -39,10 +39,17 @@ allprojects {
     repositories {
         mavenCentral()
         maven {
-            url 'https://artifactory.xenit.eu/artifactory/libs-release'
+            url "https://repo.xenit.eu/K8TBkpYRAtTc9flA/private/maven/"
```

Comment:
What is this Repo? I can't seem to access it, the URL is a hash?

Reply:
that's the cloudsmith maven repo. Im not surprised they dont allow pure http browsing. the url is generated on their webui.

Reply:
understood, thanks a lot for the expl :)